12_glob_lab/globlab.txt

```diff
@@ -5,25 +5,25 @@ https://www.tocode.co.il/bundles/unix1/lessons/globlab
 
   1. All files whose name starts with a lowercase letter
   Answer: 
-
+	ls [a-z]*
   2. All files whose name starts with an uppercase letter
   Answer:
-
+	ls [A-Z]*
   3. All files whose name ends in an "a", "e", "i", "o" or "u"
   Answer:
-
+	ls *[aeio] [u]
   4. All files whose name DOES NOT end in an "a", "e", "i", "o" or "u"
   Answer:
-
+	ls *[!aeio] [!u]
   5. All files with extension ".c" that are in a folder named "src"
   Answer:
-
+	ls /src *.c
   6. All files with extension ".c" under any folder
   Answer:
-
+	ls *.c
 2. Write a command that would delete all files ending with a ".png" under any subdirectory.
-
+	rm *.png -r
 3. The command "ls -l" can take the name of a file or directory and show details on them. From within "wildcards" folder run "ls -l [A-Z]*". What was printed? Why?
-
-4. Search in "man ls" for an option to show only directory data so the command "ls _____ D*" should print only the word Doc
+	list of all files that contain Any single upper-case ascii alphabet letter, and all the rest of files
```

Comment:
all files that begin with *any* letter except of lower case "a" or all files that begin will an upper-case letter depends on your locale, [I try to explain here](https://github.com/tocodeil/unix1-course-examples/issues/15)